lib/wsdl.js

```diff
@@ -1094,6 +1095,19 @@ WSDL.prototype._initializeOptions = function (options) {
     this.options.ignoredNamespaces = this.ignoredNamespaces;
   }
 
+  var ignoredSchemaNamespaces = options ? options.ignoredSchemaNamespaces : null;
+
+  if (ignoredSchemaNamespaces &&
+    (Array.isArray(ignoredSchemaNamespaces.namespaces) || typeof ignoredSchemaNamespaces.namespaces === 'string')) {
+    if (ignoredNamespaces.override) {
+      this.options.ignoredSchemaNamespaces = ignoredSchemaNamespaces.namespaces;
+    } else {
+      this.options.ignoredSchemaNamespaces = this.ignoredSchemaNamespaces.concat(ignoredSchemaNamespaces.namespaces);
+    }
+  } else {
+    this.options.ignoredSchemaNamespaces = this.ignoredSchemaNamespaces;
```

Comment:
Again, shouldn't it read `this.options.ignoredSchemaNamespaces = this._ignoredSchemaNamespaces`?